### PR TITLE
Fix/phpunit8.5 upgrade

### PR DESCRIPTION
**Pull request by mike-ionut-mihai-sandu-tao**
updated all test to phpunit 8.5 beacuse they were failling

### CODE REVIEW

File: test/SearchTest/UsableTait/OptionsTraitTest.php [hunk 1]
```diff
@@ -20,27 +20,33 @@
...
+{
+
     protected $instance;
-    
-    public function setup() {
-        
+
+    public function setup(): void
+    {
+
```
Comment (gabrielfs7) [inline test/SearchTest/UsableTait/OptionsTraitTest.php]: Can we remove some extra lines inserted in the files?

**Review by gabrielfs7 — APPROVED** [review]
No prod code touched, just would do some formatting adjustments and I discussed with @mike-ionut-mihai-sandu-tao

**Comment by gabrielfs7** [comment]
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are

File: test/SearchTest/UsableTait/OptionsTraitTest.php (hunk 1 above)
Comment (mike-ionut-mihai-sandu-tao) [reply test/SearchTest/UsableTait/OptionsTraitTest.php]: done

**Review by gabrielfs7 — APPROVED** [review]
lgtm

**Review by siwane — APPROVED** [review]

**Review by gabrielfs7 — APPROVED** [review]

**Review by gabrielfs7 — APPROVED** [review]

**Pull request merged**